packages/segmented-control/src/SegmentedControl.styles.ts

```diff
@@ -0,0 +1,168 @@
+import { css, cx } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import { palette } from '@leafygreen-ui/palette';
+import { fontFamilies } from '@leafygreen-ui/tokens';
+import { transparentize } from 'polished';
+import { Size } from './types';
+
+/**
+ * Styles
+ */
+export const wrapperStyle = css`
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  z-index: 0;
+  font-family: ${fontFamilies.default};
+`;
+
+export const labelStyle = css`
+  white-space: nowrap;
+`;
+
+export const labelThemeStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    letter-spacing: 1.4px;
+    color: ${palette.gray.dark1};
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.gray.light1};
+  `,
+};
+
+export const optionsWrapperStyleSize: Record<Size, string> = {
+  [Size.Small]: css`
+    --segment-gap: 1px; // space between segments
+    --wrapper-padding: 0px;
+    --indicator-height: 100%;
+  `,
+  [Size.Default]: css`
+    --segment-gap: 5px; // space between segments
+    --indicator-height: calc(100% - 2 * var(--wrapper-padding));
+  `,
+  [Size.Large]: css`
+    --segment-gap: 5px; // space between segments
+    --indicator-height: calc(100% - 2 * var(--wrapper-padding));
+  `,
+};
+
+export const optionsWrapperStyleTheme: Record<Theme, string> = {
+  [Theme.Light]: css`
+    --background-color: ${palette.gray.light3};
+    --border-width: 0px;
+    --inner-shadow: 0px 1px 2px ${transparentize(0.7, palette.black)} inset;
+    --outer-shadow: 0px 1px 1px ${palette.gray.light2};
+    --hover-background-color: ${palette.white};
+    --indicator-background-color: ${palette.black};
+    --indicator-border-color: ${palette.black};
+  `,
+  [Theme.Dark]: css`
+    --background-color: ${palette.gray.dark4};
+    --border-color: rgba(255, 255, 255, 0);
+    --border-width: 1px;
+    --inner-shadow: 0px 0px 0px 1px ${palette.gray.dark1} inset;
+    --outer-shadow: 0px 0px 0px 0px rgba(255, 255, 255, 0);
+    --hover-background-color: ${palette.gray.dark3};
+    --indicator-background-color: ${palette.gray.light2};
+    --indicator-border-color: ${palette.gray.light2};
+  `,
+};
+
+export const optionWrapperRadiusStyle: Record<Theme, Record<Size, string>> = {
+  [Theme.Light]: {
+    [Size.Small]: css`
+      --outer-radius: 6px;
+      --indicator-radius: 6px;
+    `,
+    [Size.Default]: css`
+      --wrapper-padding: 3px;
+      --outer-radius: 8px;
+      --indicator-radius: 6px;
+    `,
+    [Size.Large]: css`
+      --outer-radius: 9px;
+      --wrapper-padding: 3px;
+      --indicator-radius: 6px;
+    `,
+  },
+  [Theme.Dark]: {
+    [Size.Small]: css`
+      --outer-radius: 4px;
+      --indicator-radius: 6px;
+    `,
+    [Size.Default]: css`
+      --wrapper-padding: 2px;
+      --outer-radius: 6px;
+      --indicator-radius: 4px;
+    `,
+    [Size.Large]: css`
+      --wrapper-padding: 2px;
+      --outer-radius: 6px;
+      --indicator-radius: 4px;
+    `,
+  },
+};
```

Comment:
These shouldn't be different between Light & Dark mode anymore

Reply:
Hmm that they are... We don't do that anywhere else though. Might be worth double checking with design?

Reply:
updated!